wbooks/tsconfig.json

```diff
@@ -57,7 +57,18 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     "paths": {
       "@app/*": ["src/app/*"],
-      "@constants/*": ["src/constants/*"]
+      "@constants/*": ["src/constants/*"],
+      "'@app'" : ["./src/app"],
```

Comment:
this exist, config others same to example

Reply:
and remove simple quote

Reply:
Reviewed, thanks! Upload it again tomorrow